content/_releases/22/52.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Product Update - v22.52
+description: Details of the product update v22.52 from 29th of December 2022.
+layout: rel
+version: "22.52"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-12-29
+redirect_from:
+  - /releases/22.52.html
+---
+
+## Features
+
+### Additing tenant roles
+
+We have changed the name of permission `tenants.tenant.edit_roles` to `global.tenant.edit_roles`.
+
+Now `PATCH /v2/tenants/:tenantId/roles` allows to create a new role with permissions in a tenant scope. We have also updatet API endpoint `PATCH /v2/tenants/{tenant_id}/members/{user_id}` which allows to grant any role from tenants scope to any tenant user. This will allow to give any user `tenants.user.generate_one_time_token` permission.
+
+### Abbility to read recipies by SERVICE_ACCOUNT
+
+We try to make working with recipes as convenient as possible for our users. In a recent release [22.48](/releases/22/48#initial-setup), we have already begun some improvements in this direction. Now we are introducing a new permission `GLOBAL.RECIPE.GET` to view **all** recipes based on permissions's scope and gave the new permission to `SERVICE_ACCOUNT` with Global level. This is the first step in the development of a new feature, in which the new role of the superuser will allow you to review and approve recipes, as well as increase their level up to tenant or global if necessary.
```

Comment:
Our goal is to make easy-to-use recipes. In a recent release [22.48](/releases/22/48#initial-setup), we have already begun some improvements in this direction. Now we are introducing a new permission `GLOBAL.RECIPE.GET` to view **all** recipes based on the permission's scope and provide the new permission to `SERVICE_ACCOUNT` with the Global level. This is the first step in the development of a new feature, in which the new role of the superuser will allow you to review and approve recipes, as well as to change their level up to tenant or global if necessary.